api-reference/10 UI Components/dxChart/5 Series Types/CommonSeries/hoverStyle/dashStyle.md

```diff
@@ -5,7 +5,7 @@ default: 'solid'
 ---
 ---
 ##### shortDescription
-Specifies the dash style of the series line when the series is in the hovered state. Applies only to line-like series.
+Specifies the dash style of the series line when the series is in hovered state. Applies only to line-like series.
```

Comment:
Why did you remove 'the' here?

Reply:
Because Vladimir Abadzhev told me so.

> You don't want to use an article with words like "state", "mode", or "format".

Reply:
There is no mention of 'state' in their latest guideline on articles